### PR TITLE
FOUNDRYUI-40 Add jest-coverage-badges module

### PR DESCRIPTION
**Pull request by wmangimelli**

### CODE REVIEW

File: packages/hs-react-ui/package.json [hunk 1]
```diff
@@ -98,6 +98,7 @@
     "@types/lodash.debounce": "^4.0.6",
     "@typescript-eslint/eslint-plugin": "^3.0.0",
     "@typescript-eslint/parser": "^3.0.0",
+    "jest-coverage-badges": "^1.1.2",
```
Comment (donaldjbrady) [inline packages/hs-react-ui/package.json:101]: Can we make this a dev dependency?

File: packages/hs-react-ui/package.json [hunk 2]
```diff
@@ -123,7 +124,9 @@
     "webpack": "webpack --config webpack.config.js",
     "dts": "tsc --declaration --outDir dist/types/ --emitDeclarationOnly --declarationMap --allowJs false --checkJs false",
     "pre-push": "yarn run lint-ci && yarn run test && yarn run clean && yarn run webpack && yarn run dts",
-    "test": "rm -rf ./coverage/* && jest --collectCoverage=true",
+    "test": "rm -rf ./coverage/* && jest",
```
Comment (donaldjbrady) [inline packages/hs-react-ui/package.json:127]: `Test` should run both `test:coverage` and `test:badges` for our build process. The `test` script gets called during our CICD pipeline

File: packages/hs-react-ui/package.json [hunk 3]
```diff
@@ -123,7 +124,9 @@
...
+    "test:coverage": "npm test -- --coverage",
```
Comment (donaldjbrady) [inline packages/hs-react-ui/package.json:128]: I think you have an extra `--` here

File: packages/hs-react-ui/package.json (hunk 1 above)
Comment (wmangimelli) [reply packages/hs-react-ui/package.json:101]: Thanks for catching this.

File: packages/hs-react-ui/package.json (hunk 2 above)
Comment (wmangimelli) [reply packages/hs-react-ui/package.json:127]: I will append `&& jest-coverage-badges` to the **_test_** script so the badges are updated in the CI pipeline.

File: packages/hs-react-ui/package.json (hunk 3 above)
Comment (wmangimelli) [reply packages/hs-react-ui/package.json:128]: Removed

**Pull request merged**